README.md

```diff
@@ -198,13 +198,9 @@ For a complete list of commands, see the
 ## Building the CLI
 ### Developing
```

Comment:
What do you think of renaming the titles:
* Building the CLI --> "Contributing to CLI"
* Developing --> "Setup workspace"